vmware/live-docs
# Ng11 upgrade

Upgrading to latest Angular 11 and Clarity v5

## Steps (and Testing Done)
* Update stackblitz template to use v11 based off https://stackblitz.com/edit/clarity-light-theme-v5
  * Run app
  * Make sure clicking on the play button opens a working stackblitz
* `ng update @angular/core@10 @angular/cli@10; ng update @clr/angular@4` 
  * Update deps using versions from https://github.com/vmware/vmware-cloud-director-ui-components/blob/master/package.json 
  * Run App
  * Make sure clicking on the play button opens a working stackblitz
* `ng update @angular/core@11 @angular/cli@11; ng update @clr/angular@5`
  * Fix peer dependencies, used the versions from our internal gitlab vcd_ui repo as a guide.
    * One peer dep could not be fixed: `npm WARN tsickle@0.39.1 requires a peer of typescript@~3.9.5 but none is installed. You must install peer dependencies yourself.` Could not find a version that used our existing version of TS (4.0). 
  * Remove TS Lint and use eslint. See 
  * Run App
  * Make sure clicking on the play button opens a working stackblitz
